website/project/model.py

```diff
@@ -3729,3 +3810,125 @@ def _on_reject(self, user, token):
             },
             auth=Auth(user),
         )
+
+
+class DraftRegistrationApproval(Sanction):
+
+    mode = Sanction.ANY
+
+    def _on_complete(self, user, token):
```

Comment:
Even though this method doesn't actually do anything, it's odd that the call signature doesn't match the parent method. (see L3303)